repo/config/init.go

```diff
@@ -226,34 +218,6 @@ func defaultDatastoreConfig() native.Datastore {
 	}
 }
 
-// identityConfig initializes a new identity.
-func identityConfig() (native.Identity, error) {
```

Comment:
the old way in which the IPFS identity (keys) were generated, now moved to a higher level because we need to be able to control the input seed (mnemonic)